src/i18n/swg-strings.js

```diff
@@ -153,6 +153,12 @@ export const SWG_I18N_STRINGS = {
     'zh-hk': '您之前已訂閱。',
     'zh-tw': '你已經訂閱了。',
   },
+  'REGWALL_REGISTER_FAILED_LANG_MAP': {
+    'en': 'Failed to register.',
```

Comment:
Did you by any chance get these texts reviewd by UX/UXW? Otherwise, please follow that up (I think you can do that in a follow up PR with translation).